lib/param_parsing/toml.go

```diff
@@ -28,15 +28,18 @@ func getParamsTOML(params Params) (Params, error) {
 
 		if viperParser.Get("bin") != nil {
 			params.CustomBinaryPath = viperParser.GetString("bin")
+			logger.Debugf("Using bin from toml file %q: %q", tomlPath, params.CustomBinaryPath)
 		}
 		if viperParser.Get("log-level") != nil {
 			params.LogLevel = viperParser.GetString("log-level")
 		}
 		if viperParser.Get("version") != nil {
 			params.Version = viperParser.GetString("version")
+			logger.Debugf("Using version from toml file %q: %q", tomlPath, params.Version)
```

Comment:
```suggestion
			logger.Debugf("Using \"version\" from %q: %q", tomlPath, params.Version)
```

Reply:
The reason I wanted to say "toml" file, was that I thought I'd seen something mentioned about possibly being able to configure the TOML location independently (and wasn't sure if this would include the filename).
So could imagine a situation in future where someone had run it in an environment where an env variable that changes the TOML file path and the user wasn't aware and the name of the TOML file in the env variable made it less obvious.

i.e.
```
TF_TOML_FILE=/src/terraform   # Set in a pipeline

# pipeline runs
tfswitch
Using "version" from /src/terraform: 1.5.4
```

But I guess, it's easy to just see what is in the location anyway :)

Reply:
> But I guess, it's easy to just see what is in the location anyway :)

Yep. That's exactly what I mean =)